packages/ui-demo/src/Table.stories.tsx

```diff
@@ -105,11 +116,15 @@ export const TableStories = {
                 <Text>Cost</Text>
               </TableHeaderCell>
             </TableHeader>
-            <TableRow>
-              <Text>Data 1 has some data in it</Text>
-              <Text>Data 2 has some longer data in it</Text>
-              <Text>Data 3 is short</Text>
-            </TableRow>
+            {rows.map((row) => (
+              <TableRow key={row}>
+                <Text key={1}>{row}</Text>
+                <Text key={2}>Data</Text>
+                <Text key={3}>Other Data</Text>
+                <Text key={row}>Data</Text>
+                <Text key={row}>$2.00</Text>
```

Comment:
Not super important but this last column with $2.00 isn't used

Reply:
Ah whoops, good catch!